Jenkinsfile-new

```diff
@@ -0,0 +1,301 @@
+import org.jenkinsci.plugins.workflow.steps.FlowInterruptedException
```

Comment:
What is this file?

Reply:
It for catching and handle  `FlowInterruptedException`, If CI was Aborted